packages/shell-server/shell-server.js

```diff
@@ -441,7 +441,18 @@ function evalCommand(command, context, filename, callback) {
   }
 
   evalCommandPromise.then(function () {
-    callback(null, script.runInThisContext());
+
+    if (repl.input) {
+      callback(null, script.runInThisContext());
+    } else {
+      // If repl didn't start, `require` and `module` are not visible
+      // in the vm context
+      repl.require = require;
+      repl.module = module;
```

Comment:
These are the `require` and `module` functions available globally in the `shell-server.js` module. Instead, we should be creating a module for the REPL with `meteorInstall` and using the resulting `require` and `module` parameters, as we do [here](https://github.com/atd/meteor/blob/e5f4d0a84a04232e50b74cb386a36c0408c1054a/packages/shell-server/shell-server.js#L207-L228).